### ISSUE TITLE
Add feature descriptions for Idea Hub

### ISSUE THREAD

**Issue by aaemnnosttv** [google/site-kit-wp#3692]
## Feature Description

Currently Idea Hub does not declare any features that it provides. This is currently only shown when a module is about to be disconnected, but could be shown elsewhere in the future. Features are an optional piece that modules can provide when registering on the client side.

![image](https://user-images.githubusercontent.com/1621608/125059838-6d743700-e0b4-11eb-8036-309aa3aaac63.png)

Note: https://github.com/google/site-kit-wp/issues/3691 addresses the issue of the module disconnect prompt for modules that don't have features, this issue is just about adding features for Idea Hub (all other modules already declare them).

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance criteria

* Idea Hub should declare the following features:
    * `Suggestions for new topics to write about`

## Implementation Brief
* Using `assets/js/modules/idea-hub/index.js`,
	* Add the `features` key to the object which is the second parameter of the `modules.registerModule` function.
	* `features` is an array with only 1 item which is the string as per the AC.

### Test Coverage
* No new tests to be added.

### Visual Regression Changes
* N/A

## QA Brief
* Connect the Idea Hub module.
* Go to Site Kit -> Settings and under the Connected Services tab, disconnect the Idea Hub module.
* The modal that asks you to confirm the action should display the text as per the AC.

## Changelog entry

* Add feature description to Idea Hub module.

**Comment by marrrmarrr** [comment]
@aaemnnosttv There's only one feature really, I've added it to the ACs section of the issue description. 
Reassigning to you, move to IB if this works; else let me know if you have any other comments.

**Comment by marrrmarrr** [comment]
@aaemnnosttv let's shorten this to "Suggestions for new topics to write about" to solve the immediate issue. 
Would be great to change the styling of these modals so the text wraps. Not sure either why fixed item height was introduced, but it's definitely something we can alter.

**Comment by aaemnnosttv** [comment]
Thanks @marrrmarrr, I've updated the ACs accordingly. I'll open a new issue to address the changes to the wrapping for these items in the future 👍 

IB ✅

**Comment by wpdarren** [comment]
### QA Update: Pass ✅ 

Confirmed that the modal that asks you to confirm the action displays the text `Suggestions for new topics to write about`

![image](https://user-images.githubusercontent.com/73545194/125961265-3849932f-b45d-47a9-80aa-b1c97800048f.png)
